ProjectLighthouse/Helpers/ReflectionHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿#nullable enable
+using System.Reflection;
+
+namespace LBPUnion.ProjectLighthouse.Helpers;
+
+public static class ReflectionHelper
```

Comment:
This class could be merged with `CensorHelper`, something like `SanitizationHelper`?